NEWS.md

```diff
@@ -1,5 +1,33 @@
 ## 2021-02-27 v5.13.0
 
+### Static Permission Migration
+There are several changes related to migration of static (system-defined) permissions introduced in this release:
+
+* Permissions can now be renamed via a new `replaces` property.  Perm-users (Assignments) and PermissionSet sub-permissions will be updated automatically.
+  * Example:  `foo.get` replaces `foo.read`, `foo.view`.  Any users which were assigned either `foo.read` or `foo.view` would automatically be granted `foo.get`.  Permissions `foo.read` and `foo.view` would be marked deprecated.
+* When permissions that once appeared in a module descriptor are removed in a newer version of the module descriptor, they will be marked deprecated. 
+  * For now, this means the `displayName` of these permissions will be prefixed with `(deprecated) `, but the permissions will not be filtered out of any API calls. 
```

Comment:
Maybe consider switching the order of these two entries? In my view, removing a perm is probably more likely to be used. Hopefully dev will not abuse the `replace` feature.

Reply:
Updated in all three places